### PR TITLE
Add border to cards

### PR DESCRIPTION
**Pull request by dominik-zeglen**
This PR adds border to cards and changes default background color.

### Screenshots
Zrzut ekranu 2022-05-4 o 09 27 23

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-6710 Global styles update](https://app.clickup.com/t/2549495/SALEOR-6710)

File: src/theme/themes.ts [hunk 1]
```diff
@@ -45,6 +45,7 @@ export const dark: SaleorThemeColors = {
     3: "rgba(250, 250, 250, 0.6)",
     4: "rgba(250, 250, 250, 0.4)",
     5: "rgba(250, 250, 250, 0.1)",
+    6: "rgba(250, 250, 250, 0.1)",
```
Comment (Droniu) [inline src/theme/themes.ts:48]: 5 and 6 are exactly the same, what's the point of that?

File: src/theme/themes.ts [hunk 2]
```diff
@@ -102,7 +103,7 @@ export const light: SaleorThemeColors = {
   },
   autofill: "#f4f6c5",
   background: {
-    default: "#EFF5F8",
+    default: "#FAFBFA;",
```
Comment (Cloud11PL) [inline src/theme/themes.ts:106]: The `;` here and in `main` are unnecessary

**Review by Cloud11PL — CHANGES_REQUESTED** [review]

**Review by patrys — DISMISSED** [review]
Those chips have poor visual contrast, both in the attached screenshot and in the storybook.

File: src/theme/themes.ts (hunk 1 above)
Comment (witoszekdev) [reply src/theme/themes.ts:48]: This is because light theme has main[6] color but we don't have that for dark mode, so main.5 was reused instead ;)

**Comment by witoszekdev** [comment]
@patrys Yeah, I agree. No problem wel'll change that: we have the new design on our Figma and task for that in the backlog but for now I think we can merge this and change them in a separate PR

File: src/theme/createSaleorTheme/createSaleorTheme.tsx [hunk 3]
```diff
@@ -26,8 +26,8 @@ export const createTheme = (colors: SaleorThemeColors): SaleorTheme =>
       ...overrides(colors, fontFamily),
       MuiCard: {
         root: {
-          borderRadius: 8,
-          borderWidth: 0,
+          border: `1px solid ${colors.main[6]}`,
```
Comment (Droniu) [inline src/theme/createSaleorTheme/createSaleorTheme.tsx:29]: Card borderColor in darkmode is actually main[4] now (check figma). That's how I would do it
```suggestion
          border: "1px solid",
          borderColor: colors.theme === "dark" ?  colors.main[4] : colors.main[6],
```

**Review by Droniu — CHANGES_REQUESTED** [review]

**Review by Cloud11PL — APPROVED** [review]

**Review by Droniu — APPROVED** [review]

**Review by patrys — APPROVED** [review]

**Comment by timuric** [comment]
Lets merge this

**Comment by dominik-zeglen** [comment]
Yeah please do 🙏

**Comment by andrzejewsky** [comment]
@dominik-zeglen thanks a lot for your contribution! we really appreciate it! would you mind resolving your conflicts?

**Review by andrzejewsky — APPROVED** [review]

**Pull request merged**